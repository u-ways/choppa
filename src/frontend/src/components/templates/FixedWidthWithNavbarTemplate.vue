<template>
  <div class="container-fluid w-100 m-0 p-0 mb-5 mh-100">
    <NavbarOrganism />
    <div class="container bg-success" :class="[ this.css ]">
      <slot></slot>
    </div>
  </div>
</template>

<script>
import NavbarOrganism from "@/components/organisms/NavbarOrganism";

export default {
  name: "FixedWidthWithNavbarTemplate",
  components: { NavbarOrganism },
  props: {
    css: String,
  },
};
</script>

<style scoped lang="scss">
</style>
