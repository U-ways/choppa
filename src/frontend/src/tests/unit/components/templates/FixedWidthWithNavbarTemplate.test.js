import { describe, expect, it, shallowMount } from "@/tests/unit/test-imports";
import FixedWidthWithNavbarTemplate from "@/_old/components/templates/FixedWidthWithNavbarTemplate";

describe("Fixed Width With Navbar Template test", () => {
  it("Renders correctly", async () => {
    const wrapper = shallowMount(FixedWidthWithNavbarTemplate, {
      slots: {
        default: "Fixed Width With Navbar Template Test Slot",
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
