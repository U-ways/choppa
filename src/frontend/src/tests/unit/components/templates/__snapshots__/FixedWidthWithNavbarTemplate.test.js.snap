// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixed Width With Navbar Template test Renders correctly 1`] = `
"<div class=\\"container-fluid w-100 m-0 p-0 mb-5 mh-100\\">
  <navbarorganism-stub></navbarorganism-stub>
  <div class=\\"container\\">Fixed Width With Navbar Template Test Slot</div>
</div>"
`;
